### PR TITLE
mbeliaev/repo stat

### LINKED ISSUE #288: Support FileStatistics in Artifact stat
[FileStatistics](https://www.jfrog.com/confluence/display/JFROG/Artifactory+REST+API#ArtifactoryRESTAPI-FileStatistics)

Could you please assign this issue to me?

### PR DESCRIPTION
**Pull request by beliaev-maksim**
closes #288 

- add repo name into file stats. Since repo name from API might be different, if repo is a cached one
- added download file stats

### CODE REVIEW

File: artifactory.py [hunk 1]
```diff
@@ -862,6 +883,25 @@ def stat(self, pathobj):
...
             children=children,
+            repo=jsn.get("repo", None),
+        )
+
+        return stat
+
+    def download_stats(self, pathobj):
+        jsn = self.get_stat_json(pathobj, key="stats")
+
+        # divide timestamp by 1000 since it is provided in ms
```
Comment (allburov) [inline artifactory.py:894]: Why do we need to divide it?

File: artifactory.py [hunk 2]
```diff
@@ -862,6 +883,25 @@ def stat(self, pathobj):
             md5=checksums.get("md5", None),
             is_dir=is_dir,
             children=children,
+            repo=jsn.get("repo", None),
+        )
+
+        return stat
+
+    def download_stats(self, pathobj):
```
Comment (allburov) [inline artifactory.py:891]: `download_stats` look like "to download statistic" when it's placed near by `get_stat_json`

Let's call it `get_download_stat`

File: artifactory.py (hunk 1 above)
Comment (beliaev-maksim) [reply artifactory.py:894]: Otherwise parser cannot parse date properly

Comment (allburov) [reply artifactory.py:894]: Oh, I see

File: artifactory.py (hunk 2 above)
Comment (beliaev-maksim) [reply artifactory.py:891]: Makes, sense, let me do it

Comment (allburov) [reply artifactory.py:891]: I see, it's the way how we call `stats`, it's fine

**Pull request merged**